src/C-interface/bml_inverse.c

```diff
@@ -0,0 +1,30 @@
+#include "bml_inverse.h"
+#include "bml_introspection.h"
+#include "bml_utilities.h"
+#include "bml_logger.h"
+#include "bml_types.h"
+#include "dense/bml_inverse_dense.h"
+#include "ellpack/bml_inverse_ellpack.h"
+
+void
+bml_inverse(
+    const bml_matrix_t * A,
+    bml_matrix_t * B)
```

Comment:
I think it would be clearer if the interface was `bml_matrix_t * bml_inverse(const bml_matrix_t * A)` to indicate that a new matrix is calculated and allocated.